code/modules/admin/verbs/adminhelp.dm

```diff
@@ -590,13 +590,13 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 		send2otherserver(source,final)
 
 /// Sends a message to other servers.
-/proc/send2otherserver(source, msg, type = "Ahelp", target_servers, list/additional_data)
+/proc/send2otherserver(source, msg, type = "Ahelp", target_servers, list/additional_data = list())
 	if(!CONFIG_GET(string/comms_key))
 		debug_world_log("Server cross-comms message not sent for lack of configured key")
 		return
 
 	var/our_id = CONFIG_GET(string/cross_comms_name)
-	var/list/message = additional_data != null ? additional_data : list()
+	var/list/message = additional_data
```

Comment:
Looking now, I guess at this point you don't really need this `message` thing at all, no? Just rename `additional_data` to `message`?

This is my only concern with the code, everything else looks fine.

Reply:
mans got a point, i was hesitant to remove it to improve legibility or rather to avoid confusion but you know what helps that you are right jared fogle a documentation moment